OrbitGl/ThreadTrack.cpp

```diff
@@ -127,7 +127,9 @@ void ThreadTrack::SetTimesliceText(const Timer& timer, double elapsed_us,
       text_box->SetText(
           SystraceManager::Get().GetFunctionName(timer.m_FunctionAddress));
     } else {
-      CHECK(false);
+      ERROR("Unexpected case in ThreadTrack::SetTimesliceText");
+      PRINT_VAR(timer.m_Type);
+      PRINT_VAR(func);
```

Comment:
sorry for late review.. I just want to make sure this is not the case where we should fail because nobody knows this why is this happening.

In some cases it is better to fail fast then to continue execution in some unknown and probably corrupt state.

Reply:
I'm honestly not sure. I don't think this is a bad state per se, but I also don't know why this is happening. An alternative here would be to add the logging but also keep the CHECK(false).

Reply:
@pierricgimmig @dpallotti Any idea what case we might run in here?

Reply:
Discussed with @pierricgimmig over VC. We don't think anything bad can happen if we don't abort here. The logging should help us identify additional cases we can have here.